integration/e2e_test.go

```diff
@@ -85,27 +88,27 @@ func TestSnapshotCacheSingleEnvoyAndXdsRelayServer(t *testing.T) {
 		// TODO: parametrize node-id in bootstrap files.
 		err := configv2.SetSnapshot("test-id", snapshotv2)
 		if err != nil {
-			t.Fatalf("Snapshot error %q for %+v\n", err, snapshotv2)
```

Comment:
Will logger.Error cause the test to terminate early? If not, I think we may need to leave the Fatals alone.

Reply:
The Logger Interface also has a Fatal function, I think you can implement that and use it instead of Error.

Reply:
i think its a fair thing to terminate early when error happens , no?

Reply:
changed them to fatal now